packages/menu/src/Menu/Menu.tsx

```diff
@@ -199,6 +228,7 @@ export const Menu = React.forwardRef<HTMLDivElement, MenuProps>(function Menu(
           onEntered={handlePopoverOpen}
           data-testid={LGIDs.root}
           data-lgid={LGIDs.root}
+          ref={popoverRef}
```

Comment:
why the switch?

Reply:
b/c the popover ref should be on the popover element
I'll add to the changeset